### ISSUE TITLE
aview not working after firmware update

### ISSUE THREAD

**Issue by andycam960** [aviewdevs/aview#3]
Samsung note recent firmware upgrade aview turns error at all times

**Comment by semaj11** [comment]
Yeah aview not working on my Samsung Galaxy 2 running jelly bean and my samsung galaxy tab 2.0 aswell.
